common/buildcraft/BuildCraftCore.java

```diff
@@ -364,6 +364,9 @@ public void loadConfiguration(FMLPreInitializationEvent evt) {
 		FMLCommonHandler.instance().bus().register(this);
 		MinecraftForge.EVENT_BUS.register(this);
 		MinecraftForge.EVENT_BUS.register(new BlockHighlightHandler());
+
+		OreDictionary.registerOre("chestWood", Blocks.chest);
+		OreDictionary.registerOre("craftingTableWood", Blocks.crafting_table);
```

Comment:
Double entries makes no effect. It's safe to use just OreDictionary.registerOre.